doc/nrf/introduction.rst

```diff
@@ -153,3 +156,8 @@ See :ref:`dev-model` for more information about the |NCS| code base and how to m
 
 Repositories and revisions
 **************************
+
+The following table lists all the repositories (and their respective revisions) that are included as part of |NCS| |release| release:
```

Comment:
Oh! Looks like this change did not get added to my commit.@gmarull could you fix this?
```suggestion
The following table lists all the repositories (and their respective revisions) that are included as part of |NCS| |version| release:
```